remotespark/livyclientlib/livysession.py

```diff
@@ -61,7 +62,8 @@ def __init__(self, http_client, properties, ipython_display,
         self.created_sql_context = sql_created
 
     def __str__(self):
-        return "Session id: {}\tKind: {}\tState: {}".format(self.id, self.kind, self.status)
+        return "Session id: {}\tYARN id: {}\tKind: {}\tState: {}\n\tSpark UI: {}\n\tDriver Log: {}"\
```

Comment:
This is fine for now but let's close #230 soon.  This is obviously getting unwieldy.